### PR TITLE
richNotifications Page update for mv3

### PR DESCRIPTION
**Pull request by IanStanion-google**
This PR is in conjunction with implementing an mv3 notifications API sample

Fixes https://github.com/GoogleChromeLabs/Extension-Docs/issues/85

### CODE REVIEW

File: site/en/docs/extensions/mv3/richNotifications/index.md [hunk 1]
```diff
@@ -69,22 +69,12 @@ To use this API, call the [notifications.create][3] method, passing in the notif
...
 ```
 
 The [notifications.NotificationOptions][4] must include a [notifications.TemplateType][5], which
 defines available notification details and how those details are displayed.
 
-{% Aside %}
-
-**Consider integrating with GCM!**
-
-[Keep your users informed][6] all the time, even
```
Comment (oliverdunk) [inline site/en/docs/extensions/mv3/richNotifications/index.md:82]: Should we still keep this but just link to the chrome.gcm API?

**Review by oliverdunk — APPROVED** [review]
LGTM 👍

File: site/en/docs/extensions/mv3/richNotifications/index.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ Priorities > 0 are shown for increasing duration and more high priority notifica
 displayed in the system tray.
 
 {% Aside %}
-**Platform difference:** The `code` priority does not affect the order of notifications in Chrome version 59+ on macOS.
+**Platform difference:** The priority setting does not affect the order of notifications in Chrome version 59+ on macOS.
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/richNotifications/index.md:56]:
```suggestion
**Platform difference:** The `priority` setting does not affect the order of notifications in Chrome 59 or later on macOS.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Your work looks good. I just want to do a little lateral cleanup while you're here.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**